Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "MIT OR Apache-2.0"
 
 [dependencies]
 tonic = "0.6"
-tofn = { git = "https://github.com/axelarnetwork/tofn", branch = "main"}
+tofn = { git = "https://github.com/mobilecoinofficial/tofn", branch = "basic-ed25519-support"}
```

Comment:
This will be reverted once https://github.com/axelarnetwork/tofn/pull/207 is merged